### DISCUSSION TITLE
Which OSM data is used for navigation?

### DISCUSSION THREAD

**Discussion by Helium314** [organicmaps/organicmaps#5209] [Q&A] [answered]
I'm working on an OSM editor (SCEE, fork of the more known [StreetComplete](https://github.com/streetcomplete/StreetComplete)), and currently adding some navigation-related capabilities, like turn restriction overlay and destination tagging.

Here the question came up what is actually used by routers.
Specifically I'm thinking adding dedicated quests/overlays for potentially useful edits that are currently not doable with StreetComplete, and only inconveniently using SCEE:
* traffic lights
* stop signs
* give way signs
* various ways of destination tagging: `destination`, `destinatio:lanes`, `destination:forward/backward`, `destination_sign`
* turn lanes
* maybe more things I didn't think about (already implemented: surface, smoothness, maxspeed)

I don't want to spend a lot of time e.g. on a `destination:lanes` overlay if it's not used at all.
There are also highway type and access tags, but that's a bit out of scope for such kind of editor.

**Comment by biodranik** [comment, marked as answer]
I would say that everything is important, except destination tags. They are useful, but look complex to map.
